Gemfile

```diff
@@ -92,6 +92,7 @@ group :test do
   # Use system testing [https://guides.rubyonrails.org/testing.html#system-testing]
   gem "capybara"
   gem "capybara-screenshot"
+  gem "faker"
```

Comment:
Just a heads up: PR #90 also adds Faker. So depending on who merges first, there may be a merge conflict here.